taf/updater/updater.py

```diff
@@ -654,6 +673,7 @@ def validate_repository(
             only_validate=True,
             validate_from_commit=validate_from_commit,
             excluded_target_globs=excluded_target_globs,
+            bare=bare,
```

Comment:
`validate` probably does not need this input. It is only called when the repositories are already on disk, in which case we can automatically determine if a repository is a bare repository